packages/commonwealth/client/scripts/views/components/sidebar/governance_section.tsx

```diff
@@ -175,42 +132,7 @@ export const GovernanceSection = () => {
   );
 
   const matchesProposalRoute = matchRoutes(
-    [
-      { path: '/proposals' },
-      { path: ':scope/proposals' },
-      { path: `/proposal/${ProposalType.SubstrateDemocracyProposal}` },
-      { path: `:scope/proposal/${ProposalType.SubstrateDemocracyProposal}` },
-    ],
-    location
-  );
-
-  const matchesReferendaRoute = matchRoutes(
-    [
-      { path: '/referenda' },
-      { path: ':scope/referenda' },
-      { path: `/proposal/${ProposalType.SubstrateDemocracyReferendum}` },
-      { path: `:scope/proposal/${ProposalType.SubstrateDemocracyReferendum}` },
-    ],
-    location
-  );
-
-  const matchesTreasuryRoute = matchRoutes(
-    [
-      { path: '/treasury' },
-      { path: ':scope/treasury' },
-      { path: `/proposal/${ProposalType.SubstrateTreasuryProposal}` },
-      { path: `:scope/proposal/${ProposalType.SubstrateTreasuryProposal}` },
-    ],
-    location
-  );
-
-  const matchesTipsRoute = matchRoutes(
-    [
-      { path: '/tips' },
-      { path: ':scope/tips' },
-      { path: `/proposal/${ProposalType.SubstrateTreasuryTip}` },
-      { path: `:scope/proposal/${ProposalType.SubstrateTreasuryTip}` },
-    ],
+    [{ path: '/proposals' }, { path: ':scope/proposals' }],
```

Comment:
Not sure, but does this fail on `/Proposals`, `/pRoPosAls` or any other letter case combination?

Not a blocker but just for knowledge ^

Reply:
Works with any case.